lib/find-python.js

```diff
@@ -18,8 +18,7 @@ PythonFinder.prototype = {
   log: logWithPrefix(log, 'find Python'),
   argsExecutable: [ '-c', 'import sys; print(sys.executable);' ],
   argsVersion: [ '-c', 'import sys; print("%s.%s.%s" % sys.version_info[:3]);' ],
-  semverRange: process.env.EXPERIMENTAL_NODE_GYP_PYTHON3 ? '2.7.x || >=3.5.0'
-    : '>=2.7.0 <3.0.0',
+  semverRange: '2.7.x || >=3.5.0',
```

Comment:
Do we need to enforce an upper bound?

Reply:
Not yet.

Reply:
I don't think we should. If a new version of Python breaks something here, we should fix it. If we can't fix it for some reason, then would be a good time to exclude that version.

Reply:
Ah sorry. I misread this line as 3.5.0 being an upper bound but it's the lower bound for Python 3.